sampledocs/docs/index.md

```diff
@@ -236,3 +234,18 @@ zenuml
     Alice->John: See you later!
 ```
 
+### Architecture Beta
+
+```
+architecture-beta
```

Comment:
This requires also an update of the diagram detection code https://github.com/johanneswuerbach/backstage-plugin-techdocs-addon-mermaid/blob/main/src/Mermaid/hooks.ts#L18, could you add this?

Reply:
yes